configure.ac

```diff
@@ -112,8 +112,18 @@ AC_DEFINE_UNQUOTED([UCX_CONFIGURE_FLAGS], ["$config_flags"], [UCX configure flag
 #
 # Define path of ucx.conf configuration file
 #
+case "$prefix" in
+    NONE)
+        case "$sysconfdir" in
+            '${prefix}/etc')
+                sysconfdir=/etc # Prevent sysconfdir from being "NONE/etc"
+                ;;
+        esac
+        ;;
+esac
+# Evaluate the '${prefix}/etc' expression
 SYSCONFDIR="`eval echo $sysconfdir`"
-AC_SUBST([ucx_conf_file], [${SYSCONFDIR}/ucx.conf])
+AC_SUBST([ucx_conf_file], [${SYSCONFDIR}/ucx/ucx.conf])
```

Comment:
maybe set ${ucx_conf_dir}
and the filename itself could be defined in UCX parser code